### PR TITLE
removed redundant header include and error check print

### PR DESCRIPTION
**Pull request by sarab96**
@maxzhen Please do the needful

### CODE REVIEW

**Comment by maxzhen** [comment]
ok to test

**Comment by sonals** [comment]
retest this please

**Comment by sarab96** [comment]
@maxzhen Please merge this

**Comment by maxzhen** [comment]
The pipeline seems to take too long to finish for this PR? Could someone take a look?

**Comment by maxzhen** [comment]
retest this please

File: src/xma/src/xmaplugin/xmaplugin.cpp [hunk 1]
```diff
@@ -43,7 +43,10 @@ xma_plg_buffer_alloc(XmaHwSession s_handle, size_t size)
 #if 0
     printf("xma_plg_buffer_alloc handle = %d\n", handle);
 #endif
-
+  
+    if (handle <= 0) {
```
Comment (sonals) [inline src/xma/src/xmaplugin/xmaplugin.cpp]: I thought 0 is a valid handle?

Comment (sarab96) [reply src/xma/src/xmaplugin/xmaplugin.cpp]: yes you are correct. Made update.

**Comment by sarab96** [comment]
@sonals Please approve and merge

**Pull request merged**